Datasnap/Classes/Utilities/Datasnap.m

```diff
@@ -77,45 +83,45 @@ - (id)initWithApiKey:(NSString*)apiKey
         apiKeySecret:(NSString*)apiKeySecret
       organizationId:(NSString*)organizationId
            projectId:(NSString*)projectId
-      eventQueueSize:(NSInteger)eventNum
+           idfaOptIn:(bool)optIn
+               email:(NSString*)email
```

Comment:
There's this custom init method with a bunch of properties and also a sharedClient method.  However, the sharedClient does not make use of the custom init...  Are there multiple Datasnap objects in the app lifecycle? Who calls the custom init?